src/frontend/screens/Onboarding/AddToProjectScreen/DeviceFoundStep.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from "react";
+import { Animated, View, StyleSheet } from "react-native";
+import { defineMessages, FormattedMessage } from "react-intl";
+import CheckBox from "@react-native-community/checkbox";
+
+import Button from "../../../sharedComponents/Button";
+import Text from "../../../sharedComponents/Text";
+import { TouchableWithoutFeedback } from "../../../sharedComponents/Touchables";
+import { MAGENTA, MAPEO_BLUE, MEDIUM_BLUE, WHITE } from "../../../lib/styles";
+
+type Role = "participant" | "coordinator";
+
+const AnimatedCheckBox = Animated.createAnimatedComponent(CheckBox);
+
+const m = defineMessages({
+  title: {
+    id: "screens.Onboarding.AddToProjectScreen.DeviceFoundStep.title",
+    defaultMessage: "Device {deviceId} Found",
+  },
+  participantOptionTitle: {
+    id:
+      "screens.Onboarding.AddToProjectScreen.DeviceFoundStep.participantOptionTitle",
+    defaultMessage: "This device is a Participant",
+  },
+  coordinatorOptionTitle: {
+    id:
+      "screens.Onboarding.AddToProjectScreen.DeviceFoundStep.coordinatorOptionTitle",
+    defaultMessage: "This device is a Coordinator",
+  },
+  coordinatorOptionDescription: {
+    id:
+      "screens.Onboarding.AddToProjectScreen.DeviceFoundStep.coordinatorOptionDescription",
+    defaultMessage: "Coordinators can add and remove devices from projects",
+  },
+  selectValidationError: {
+    id:
+      "screens.Onboarding.AddToProjectScreen.DeviceFoundStep.selectValidationError",
+    defaultMessage: "Select a role for the device",
+  },
+  cancel: {
+    id: "screens.Onboarding.AddToProjectScreen.DeviceFoundStep.cancel",
+    defaultMessage: "Cancel",
+  },
+  inviteDevice: {
+    id: "screens.Onboarding.AddToProjectScreen.inviteDevice",
+    defaultMessage: "Invite Device",
+  },
+});
+
+interface Props {
+  deviceId: string;
+  goBack: () => void;
+  goNext: () => void;
+}
+
+export const DeviceFoundStep = ({ deviceId, goBack, goNext }: Props) => {
+  const [selectedRole, setSelectedRole] = React.useState<Role>();
+  const [showError, setShowError] = React.useState(false);
+  const pulseAnimationValue = React.useRef(new Animated.Value(0)).current;
+
+  const animatedCheckBoxStyles = React.useMemo(
+    () => ({
+      transform: [
+        {
+          scale: pulseAnimationValue.interpolate({
+            inputRange: [0, 0.5, 1],
+            outputRange: [1.2, 1.5, 1.2],
+          }),
+        },
+      ],
+    }),
+    []
+  );
+
+  const createRoleOption = (role: Role) => (
+    <AnimatedCheckBox
+      onValueChange={createOptionHandler(role)}
+      style={animatedCheckBoxStyles}
+      tintColors={{ false: showError ? MAGENTA : undefined, true: MEDIUM_BLUE }}
+      value={selectedRole === role}
+    />
+  );
+
```

Comment:
I think the designs were for radio buttons? Since only one option can be checked, I think radio makes more sense?

Reply:
ah yeah they were. thought it would make sense to use checkboxes but can switch it

Reply:
fixed 👍